q2_phylogeny/_raxml.py

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import tempfile
+import subprocess
+import skbio
+
+from q2_types.feature_data import AlignedDNAFASTAFormat
+from q2_types.tree import NewickFormat
+
+
+def run_command(cmd, verbose=True, env=None):
```

Comment:
This command is already defined in `_fasttree.py` - how about we delete this duplicate function and instead import the command:

```python
from q2_phylogeny._fasttree import run_command
```

Does that make sense?

Reply:
The run command in `_raxml.py` is different than that in `_fasttree.py`. Mainly, there is not standard input / output like FastTree.